### PR TITLE
Mepts 931 tx ml change request mer 2.6

### PR DESCRIPTION
**Pull request by ItsMurumba**

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/TxMlCohortQueries.java [hunk 1]
```diff
@@ -1468,15 +1471,15 @@ public CohortDefinition getPatientsIITLessThan90DaysComposition() {
...
+            getPatientsWhoMissedNextAppointmentAndDiedDuringReportingPeriod(),
             "startDate=${startDate},endDate=${endDate},location=${location}"));
     cd.addSearch(
         "transferredOut",
         EptsReportUtils.map(
             getPatientsWhoMissedNextAppointmentAndTransferredOut(),
             "startDate=${startDate},endDate=${endDate},location=${location}"));
 
-    cd.setCompositionString("(missedAppointment AND C1) AND NOT dead AND NOT transferredOut");
+    cd.setCompositionString("(missedAppointment AND C1) AND NOT dead ANd NOT transferredOut");
```
Comment (ningosi) [inline api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/TxMlCohortQueries.java]: maybe we need to maintain the expressions key works keep it to upper case only, you have a mix here I can see **And**

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/reports/SetupMERQuarterly26.java [hunk 2]
```diff
@@ -108,18 +107,22 @@ public ReportDefinition constructReportDefinition() {
     rd.setName(getName());
     rd.setDescription(getDescription());
     rd.setParameters(txPvlsDataset.getParameters());
-    rd.addDataSetDefinition("N", Mapped.mapStraightThrough(txNewDataset.constructTxNewDataset()));
-    rd.addDataSetDefinition(
-        "C", Mapped.mapStraightThrough(txCurrDataset.constructTxCurrDataset(true)));
-    rd.addDataSetDefinition("P", Mapped.mapStraightThrough(txPvlsDataset.constructTxPvlsDatset()));
+    //    rd.addDataSetDefinition("N",
```
Comment (ningosi) [inline api/src/main/java/org/openmrs/module/eptsreports/reporting/reports/SetupMERQuarterly26.java]: Please re enable the other datasets into the report definition

**Review by ningosi — CHANGES_REQUESTED** [review]
Look at the comments

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/reports/SetupMERQuarterly26.java (hunk 2 above)
Comment (ItsMurumba) [reply api/src/main/java/org/openmrs/module/eptsreports/reporting/reports/SetupMERQuarterly26.java]: Done

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/TxMlCohortQueries.java (hunk 1 above)
Comment (ItsMurumba) [reply api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/TxMlCohortQueries.java]: Done

**Review by cfaife — APPROVED** [review]

**Review by psmatsinhe — APPROVED** [review]
Looks good to me

**Review by psmatsinhe — APPROVED** [review]
Looks good to me

**Pull request merged**